### PR TITLE
Move `CSharpUseCompoundAssignmentCodeFixProvider` to `VerifyCS`

### PR DESCRIPTION
**Pull request by DoctorKrolic**

### CODE REVIEW

**Review by CyrusNajmabadi — APPROVED** [review]

File: src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs [hunk 1]
```diff
@@ -1194,27 +1239,27 @@ int M(int i)
...
+            "M( /* Before */ i [|=|] i - 1 /* After */ );",
             "M( /* Before */ --i /* After */ );")]
         public async Task TestTriviaPreserved(string before, string after)
         {
-            await TestInRegularAndScript1Async(
+            await VerifyCS.VerifyCodeFixAsync(
 @$"public class C
 {{
-    int M(int i)
+    void M(int i)
```
Comment (CyrusNajmabadi) [inline src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs]: Not a fan of changing test content

**Comment by CyrusNajmabadi** [comment]
Thanks!

File: src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs (hunk 1 above)
Comment (DoctorKrolic) [reply src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs]: Well, it is better to change 1 word in test than to add compiler error diagnostic, which is not the main point of the test, right?

Comment (CyrusNajmabadi) [reply src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundAssignmentTests.cs]: If you are going to change the tests, please do so in another commit. Doing so in the same commit just increases the cost and complexity of reviewing here. Thanks! :-)

**Pull request merged**